### ISSUE TITLE
Not able to run rbeautify because of "Could not find 'ruby-beautify' (>= 0) "

### ISSUE THREAD

**Issue by bobber205** [Glavin001/atom-beautify#137]
Works in console with

```
ruby /Users/alexwait/.rvm/gems/ruby-1.9.3-p448/bin/rbeautify 
```

![settings_-__users_alexwait_chirpify_chirpify-frontend_-_atom](https://cloud.githubusercontent.com/assets/5841107/4923363/efec77b0-6519-11e4-9bcc-b4ddccb7c6aa.png)

Path is set in settings page of package.
Also tried restarting atom multiple times (after each step).

**Comment by suranyami** [comment]
I'm getting this, too, but tried view->reload and it still gives the same error. Is there some other config required to get RVM's ruby to be the same one Atom is using?

**Comment by Glavin001** [comment]
@suranyami: I just released v0.25.0 (and v0.26.0) which includes support for Rubocop (see #226 and #275 ) and also auto-detects environment variables from Shell. This may help you with having Atom Beautify use RVM's Ruby, as it should perform just like your Shell does, now.
